src/components/teamBoxes/teamOneBox.css

```diff
@@ -42,13 +42,16 @@
   flex-direction: column;
```

Comment:
wrap is well done! we still need to make fontsize smaller for smaller screens because im seeing this: 
<img width="405" alt="image" src="https://user-images.githubusercontent.com/103370255/219882278-b3cbc145-32c0-4805-bff4-10e8b98326a0.png">

but i think that's actually a separate ticket